CapabilityStatement/England-CapabilityStatement.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<CapabilityStatement xmlns="http://hl7.org/fhir">
+    <id value="England-CapabilityStatement" />
+    <url value="https://fhir.nhs.uk/StructureDefinition/England-CapabilityStatement" />
+    <version value="1.0.0" />
+    <name value="EnglandCapabilityStatement" />
+    <title value="England CapabilityStatement" />
+    <status value="draft" />
+    <date value="2023-06-01" />
+    <publisher value="NHS England" />
+	<contact>
```

Comment:
Needs alignment

Reply:
Will be dealt with when merging in from develop